web/src/components/Accordion.tsx

```diff
@@ -1,30 +1,46 @@
 import React, { useRef, useEffect, useState, ReactElement } from 'react'
 import styled from 'styled-components'
 
-const AccordionWrapper = styled.div<{ height: number }>`
-  overflow: hidden;
+const AccordionWrapper = styled.div<{ height: number; $overflowVisible: boolean }>`
   transition: height 0.3s ease;
   height: ${props => props.height}px;
   width: 100%;
+  overflow: ${props => (props.$overflowVisible ? 'visible' : 'hidden')};
 `
 
 type AccordionProps = {
   isOpen: boolean
+  childrenDependency?: boolean
   children: React.ReactNode
 }
 
-const Accordion = ({ isOpen, children }: AccordionProps): ReactElement => {
+const Accordion = ({ isOpen, childrenDependency = true, children }: AccordionProps): ReactElement => {
```

Comment:
What does this childrenDependency do? It doesn't look to me like we need it.

Reply:
In this case I don't need the children to change the height when they change...

Reply:
I think this is a premature optimization here, we're only using the Accordion in one place, and there we don't use this case, so we can remove that for now. If we need it later on, we can add it then.

Reply:
Premature optimization here? sound like a good thing.. right?

Reply:
Hehe, no, it doesn't. It's optimizing for a potential future that may never happen. Or if it does happen, it may have completely different requirements and we'll need to rewrite the entire thing. In general, it's not the best idea to build in edge cases because they might exist in the future at some point.

That aside, it is not obvious from reading the code (at least to me) what it does, so it would need at least a different name. But as I said, I don't think we need it at all.